sources/packages/web/src/views/aest/student/SINManagement.vue

```diff
@@ -131,7 +131,7 @@ export default defineComponent({
         await StudentService.shared.getStudentSINValidations(props.studentId);
     };
 
-    watch(props.studentId, loadSINValidations, { immediate: true });
+    watch(toRef(props, "studentId"), loadSINValidations, { immediate: true });
```

Comment:
what happens here

Reply:
Is it not the same as below?
```ts
watch(() => props.studentId, loadSINValidations, { immediate: true });
```

Reply:
Watch in composition api needs to receive an ref object instead of just a non reactive literal value:
https://stackoverflow.com/a/69457331

Reply:
Did you test the suggested code?

Reply:
@andrewsignori-aot the way you've suggested is better. I've changed. Thanks!